index.html

```diff
@@ -12771,30 +12773,36 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
             </caption>
             <thead>
               <tr>
-                <th scope="col">Value</th>
+                <th scope="col">Value (keyword)</th>
+                <th scope="col">State</th>
                 <th scope="col">Description</th>
               </tr>
             </thead>
             <tbody>
               <tr>
                 <th class="value-name" scope="row">false</th>
+                <th class="value-name" scope="row">False</th>
                 <td class="value-description">The element supports being checked but is not currently checked.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">mixed</th>
+                <th class="value-name" scope="row">Mixed</th>
                 <!-- make sure this value remains synced with its counterpart in #aria-pressed -->
                 <td class="value-description">Indicates a mixed mode value for a tri-state checkbox or menuitemcheckbox.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">true</th>
+                <th class="value-name" scope="row">True</th>
                 <td class="value-description">The element is checked.</td>
               </tr>
               <tr>
-                <th class="value-name" scope="row"><strong class="default">undefined</strong> (default)</th>
+                <th class="value-name" scope="row"><strong class="default">undefined</strong></th>
+                <th class="value-name" scope="row">Undefined</th>
                 <td class="value-description">The element does not support being checked.</td>
               </tr>
             </tbody>
           </table>
+          <p>The attribute's <a data-cite="html/common-microsyntaxes.html#missing-value-default">missing value default</a> and <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> are both the Undefined state.</p>
```

Comment:
Migrated comment from @keithamus ([original](https://github.com/w3c/aria/pull/2413/files#r1983923427)):

I don't think we want the missing value default here to return the Undefined state, that will mean the `.ariaChecked` property will return `"undefined"` when `aria-checked` isn't on the element. I presume we want to return `null` instead? Browsers currently return null, so missing should be "no state", i.e. removed:

```suggestion
			<p>The attribute has no <a data-cite="html/common-microsyntaxes.html#missing-value-default">missing value default</a> and its <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> are both the Undefined state.</p>
```

Reply:
@keithamus Updated "Undefined state" to reference that it is equivalent to null.